client/signup/jetpack-connect/login-or-signup.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
```

Comment:
since this file is not being used anywhere yet, would it be ok to let it out of this PR? (just for the sake of keeping it smaller)